jepsen/scalardb/project.clj

```diff
@@ -7,6 +7,6 @@
                  [jepsen "0.1.10-SNAPSHOT"]
                  [cassandra "0.1.0-SNAPSHOT"]
                  [clojurewerkz/cassaforte "3.0.0-alpha2-SNAPSHOT"]
-                 [com.scalar-labs/scalardb "1.0.0-rc1" :exclusions [org.slf4j/slf4j-log4j12]]]
+                 [com.scalar-labs/scalardb "1.0.0" :exclusions [org.slf4j/slf4j-log4j12]]]
```

Comment:
Regarding this kind of version numbers (also for C*), can they be defined outside of the programs instead of hard-coding them in the code ?

Reply:
`project.clj` is like `build.gradle`.
We can change only this file to define dependent libraries.

Reply:
Got it ! Thanks